### ISSUE TITLE
Update WCAG tags for consistency with ACT rules

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3458]
The following rules have tags that are inconsistent with the ACT requirements mapping. I would like to have these changes made to improve our consistency with ACT.

- [ ] `label`: Remove `wcag131`
- [ ] `select-name`: Remove `wcag131`
- [ ] `aria-hidden-focus` remove `wcag131`
- [ ] `input-image-alt` Add `wcag412`
- [ ] `area-alt`: Remove `wcag111`

Because of how the changelog gets generated, I we can not do them all in one pull request. We should have the `wcag131` changes in one pull request, and then the other two changes in separate pull requests.

**Comment by WilcoFiers** [comment]
Nobody pick this up yet please. I need to have a few conversations before we take this on.

**Comment by WilcoFiers** [comment]
These are all agreed on. We're ready to put these in.
